app/views/reviews/_review.html.haml

```diff
@@ -0,0 +1,32 @@
+%div= render 'reviews/form'
+
+- if !@reviews.nil?
+  %ul.clearfix#review_block
+    - @reviews.each do |review|
+      -if !review.title.empty?
+        .review-container{id: "review_id-#{review.id}"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace